oat-sa/extension-tao-proctoring
# Fix/TR-2812/session history date filter

Related to https://oat-sa.atlassian.net/browse/TR-2812

Frontend sends date in a format that backend cannot parse.
So let's explicitly specify communication format and convert value visible to the user to the format understood by the server.


How to test:
- looks like there are no unit tests for this 
- enable `taoProctoring` extension
- create a test delivery with "_Require Proctoring_" on, then run this test as a test-taker
- then create a user with `Proctor` role, then login as this user
- you should see your test, press "_Monitor_"; if necessary, change "_Started at_" filter in the table; click button in the last column
- follow steps in the ticket - choose dates in the date filter and press '_Filter_'.
![Screenshot_13](https://user-images.githubusercontent.com/38751932/143398864-bacfabb6-2eb9-444f-86e4-79e53e78344b.png)

![Screenshot_14](https://user-images.githubusercontent.com/38751932/143398883-d5a88b96-994d-4860-ba97-22f8e9b1de42.png)

![Screenshot_15](https://user-images.githubusercontent.com/38751932/143399400-129956bf-c5f1-42ae-94e1-0a4ad19aa07b.png)
